lib/shopify-cli/messages/messages.rb

```diff
@@ -267,7 +267,7 @@ module Messages
           DEVELOPMENT
 
           shell_shim: <<~MESSAGE,
-          {{x}} This version of Shopify App CLI is no longer supported. You’ll need to upgrade to continue using it. This process typically takes a few minutes.
+          {{x}} This version of Shopify App CLI is no longer supported. You’ll need to upgrade to continue using it.
 
             Please visit this page for complete instructions:
             {{underline:https://shopify.github.io/shopify-app-cli/upgrade/}}
```

Comment:
```suggestion
            {{underline:https://shopify.github.io/shopify-app-cli/migrate/}}
```

@paulomarg I'll leave it to you to make the change the repo handle, if that's what we end up doing...

Reply:
Yeah that sounds good. Renaming the repo should be its own PR in any case as we'll definitely want to check for other references to it.